src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -310,6 +310,10 @@
 	top: 2px;
 }
 
+.interactive-item-container .value .rendered-markdown .chat-codeblock-pill-widget .codicon {
+	top: 0px;
```

Comment:
Sorry! You made me realize that I messed up codicons in toolbars too. We don't have a good way to select "normal markdown text". Or to center the icons in a natural way without screwing up other ones. As far as I can tell this whole problem is just from the codicon font laying out characters in a slightly weird way.